app/views/users/show.html.erb

```diff
@@ -2,6 +2,7 @@
   <h1> <%= current_user.first_name %>'s Dashboard </h1>
 
   <%= button_to 'Log Out', logout_path, method: 'delete', class: "btn btn-primary mb1 bg-teal" %>
+  <button class = "btn btn-primary mb1 bg-teal"><%= link_to "Connect to GitHub", "/auth/github" %></button>
```

Comment:
User will have to login and auth github.
Avoids passing in token or specific key

Reply:
Is this the link that you said connects to my github for some reason?

Reply:
I changed it to "/auth/github". It was originally linked to your github token just so I can verify it clicks to something else but now it shouldn't. Everything still linked to your account is under GitHubService controller because the API stuff is still linked with your github token. Haven't figured out how to refactor it so that wouldn't happen.